test/serializers/attributes_test.rb

```diff
@@ -23,12 +23,6 @@ def test_attributes_with_fields_option
                      @profile_serializer.attributes(fields: [:name]))
       end
 
-      def test_required_fields
-        assert_equal({name: 'Name 1', description: 'Description 1'},
-                     @profile_serializer.attributes(fields: [:name, :description], required_fields: [:name]))
-
-      end
-
```

Comment:
This makes it look like you could set `:name` as a required_field.  Is that still possible?